pages/linux/arch-chroot.md

```diff
@@ -0,0 +1,28 @@
+# arch-chroot
+
+> Enhanced `chroot` command for Arch Linux installation.
+> More information: <https://man.archlinux.org/man/arch-chroot.8>.
+
+- Start interactive shell in a new root directory:
+
+`arch-chroot {{path/to/new/root}}`
+
+- Specify user to use:
+
+`arch-chroot -u {{user}} {{path/to/new/root}}`
+
+- Run command in a new root directory:
+
+`arch-chroot {{path/to/new/root}} {{command}} {{command_arguments}}`
+
+- Generate the initial ramdisk using the preset provided by the `linux` package on new root directory:
+
+`arch-chroot {{path/to/new/root}} {{mkinitcpio --preset linux}}`
+
+- Start a sh session in a new root directory:
```

Comment:
To me this and the next example seem unecessary; I'd say the first example covers all of the functionality demonstrated here.  What do you think?

(also `a sh` is incorrect, at least for me, as I read 'sh' as the sequential letters 's' 'h')

Reply:
I'm not sure about `sh` but the tip about install `zsh` first is good.